web/src/hooks/useAxiosQuery.ts

```diff
@@ -76,7 +79,8 @@ export function useAxiosQuery<TData = unknown>(url: string, options?: UseAxiosQu
   }, [res.data, url])
 }
 
-/** Make multiple cached fetches in parallel (and uses `Suspense`, by contrast to the default `useQueries()`)  */
+/** Make multiple cached fetches in parallel */
+// TODO: write a test this function, as it is currently unused
```

Comment:
```suggestion
// TODO: write a test for this function, as it is currently unused
```

Why do we need the test and the function when it's actually unused?